packages/loopring_v3/contracts/impl/libexchange/ExchangeBlocks.sol

```diff
@@ -67,7 +67,7 @@ library ExchangeBlocks
         bytes  memory data,
         bytes  memory /*offchainData*/
         )
-        internal  // inline call
+        public
```

Comment:
It's important that `commitBlock` remains `internal` for efficiency, otherwise instead of passing all the DA data by reference we need to copy the data again. This costs quite a bit of gas even when using assembly, and this costs a lot of gas with the copy code generated by solidity.

Better to make another function `public` or decrease the `run` parameter of optimization if the deployment cost is still too high (if this is still the case with `cloning`).

Reply:
unfortunately, making the method `internal` and reducing the run to `1` still end up more-than-block-gas-limit error.

Reply:
let me try to fix this later in a separate PR.

Reply:
Actually, I figured out a way to fix it.